homeassistant/components/dynalite/__init__.py

```diff
@@ -251,6 +249,46 @@ async def dynalite_service(service_call: ServiceCall):
         ),
     )
 
+    def websocket_get_config_entry_data(hass, connection, msg):
+        """Get the data of a config entry."""
+        connection.send_result(
+            msg["id"],
+            {"data": dict(hass.config_entries.async_get_entry(msg["entry_id"]).data)},
+        )
+
+    ws_type_dynalite_get_entry = "dynalite/get_entry"
+    schema_websocket_get_entry = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend(
```

Comment:
This pattern is legacy. Please use the decorator pattern instead. See eg the ozw integration websocket api module.

Reply:
ok. just pushed this. please let me know if i understood it correctly